tests/runtime/builtin

```diff
@@ -108,6 +108,15 @@ EXPECT SUCCESS .*
 AFTER ./testprogs/uprobe_negative_retval
 TIMEOUT 5
 
+# Disabled, since BCC code it depends on is prone to race condition
+NAME func_uprobe_exited_process
+ENV BPFTRACE_CACHE_USER_SYMBOLS=PER_PID
+PROG uprobe:./testprogs/uprobe_symres_exited_process:test { print(func); exit(); }
+EXPECT test
+BEFORE ./testprogs/uprobe_symres_exited_process
+TIMEOUT 5
+REQUIRES bash -c "exit 1"
+
```

Comment:
Ok, makes sense. I think there'd be a race condition between the target process exiting and bpftrace printing, but it'd be so extremely unlikely to happen that it doesn't matter for this.

What's the race condition you mention in the comment about this test being disabled though?

Reply:
Just for clarification, the sequence of events that happens 99%+ of the time:
- `uprobe_symres_exited_process` starts and sleeps for 3 seconds.
- bpftrace attaches probe on `uprobe_symres_exited_process:test` and pre-loads the cache with the use of util function `get_pids_for_program` and `bcc_symcache_new`.
- the process wakes up, executes `test()` and immediately exists
- the probe fires and `bcc_symcache_resolve` is run on the stored cache (note that `symbol_table_cache_` is not used here, since `BPFTRACE_CACHE_USER_SYMBOLS` is set to `PER_PID`)

Now what happens in `bcc_symcache_resolve` is that the binary of `uprobe_symres_exited_process` is opened to look for the symbol. However, since in a general case the binary can be in a different mount namespace, BCC uses a file descriptor of `/proc/<pid>/root` that was pre-opened in `bcc_symcache_now` to access the file using `openat` (see iovisor/bcc#4319). The issue is that this approach with `openat` sometimes does not work due to current implementation issues, hence BCC checks if the process is running, and if yes, it does not use `openat`. This is prone to a race condition, since the process may exit *after* this check is done.

This race condition is much rarer than the original one from #2284 linked above (which in my case happened 100% of the time), but still happens and would cause CI randomly failing, hence the test being disabled.